meson.build

```diff
@@ -0,0 +1,10 @@
+project('vinum', 'c',
```

Comment:
add meson_version: '>= 1.1', to fix the warning bellow.
> vinumc| WARNING: Project specifies no minimum version but uses features which were added in versions:
vinumc| * 1.1: {'meson.options file'}